lib/Runtime/Library/JsBuiltInEngineInterfaceExtensionObject.cpp

```diff
@@ -297,12 +301,14 @@ namespace Js
         PropertyId methodPropID = PropertyIds::_none;
         PropertyString *methodPropString = nullptr;
         PropertyString *classPropString = nullptr;
+        JavascriptString *fullName = nullptr;
+        JavascriptString *dot = library->GetDotString();
         switch (funcKind)
         {
-#define VALUE(ClassName, methodName, _isStatic) \
-        case FunctionKind::ClassName##_##methodName: \
+#define VALUE(ClassName, methodName, _isStatic, propertyType) \
+            case FunctionKind::ClassName##_##methodName: \
```

Comment:
This case was unindented intentionally: I always hate styling switch/case in any codebase, but the Chakra codebase seems to have settled on
```c++
switch
{
case 0:
    dothingfor0();
    break;
case X:
    ...
}
```
The goal was to stick to that styling.

Reply:
Gotcha.